salt/states/user.py

```diff
@@ -359,6 +364,17 @@ def present(name,
 
         .. versionchanged:: 2015.8.0
     '''
+
+    '''
```

Comment:
Could you please prefix this comment with `#` symbols instead of using triple-quotes? This confuses our docs parser.